features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreaksFragment.kt

```diff
@@ -83,4 +98,38 @@ class FreaksFragment : Fragment() {
             repeat(FREAKS_COUNT) { this.add(freak) }
         }
     }
+
+    private fun loadFilters(activeFilter: String) =
+        if (activeFilter == "Skills") {
+            listOf<String>("Android", "Kotlin", "Other Skill", "iOS")
+        } else {
+            listOf("Freaks Catalog", "Proj2", "Tutorial", "Altkeva")
+        }
+
+    private fun showFilterModal(activeFilter: String, otherView: ViewGroup?){
```

Comment:
typo in Modal

Reply:
There is a warning over otherView. We should always make sure to resolve all this warnings android studio gives us